containers/podman-desktop-extension-bootc
# chore: rename iso to anaconda-iso and add docs

chore: rename iso to anaconda-iso and add docs

### What does this PR do?

* Changes the type from iso to anaconda-iso as ".iso" may be used in the
  future for something else, see: https://github.com/osbuild/bootc-image-builder/issues/433

### Screenshot / video of UI



N/A, backend code / use anaconda-iso option

### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-extension-bootc/issues/482

### How to test this PR?



1. Test build .iso
2. See that is succesfully creates the .iso

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
